src/IO/FileExplorer.h

```diff
@@ -16,13 +15,40 @@ class FileExplorer {
 
     FileExplorer();
 
+    /**
+     * Sets the currently visited directory to currentDir.
+     *
+     * @param currentDir
+     */
     void setCurrentDir(std::string currentDir);
+
+    /**
+     * Sets the filter for what files to keep track of in the current directory.
+     *
+     * @param filter
```

Comment:
Should have a small description of `filter`

Reply:
Done.